lib/src/default_extension_map.dart

```diff
@@ -413,6 +413,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'm3a': 'audio/mpeg',
   'm3u': 'audio/x-mpegurl',
   'm3u8': 'application/vnd.apple.mpegurl',
+  'm4a': 'audio/m4a',
```

Comment:
```suggestion
  'm4a': 'audio/mp4',
```

+1 on using `audio/mp4`. It seems to be the more standard approach.